### ISSUE TITLE
[CS 1.6] [Feature Request] cl_ignoregrenaderadio

### ISSUE THREAD

**Issue by Maxi605** [ValveSoftware/halflife#2615]
I client-side cvar to be able to ignore the radio message when a player throws a grenade.

CS:GO has this but as server-side, i don't know if is better to do it server-side or not.

**Comment by basuritashka** [comment]
It is server-side probably because CS:GO have a official competitive matchmaking and people is forced to certain settings to get a decent gameplay experience, it won't matter at all if its client-side or server-side for Counter-Strike 1.6 as there is no "official" competitive matchmaking.

**Comment by afwn90cj93201nixr2e1re** [comment]
We should add something like
cl_ignoreradio: 
0 - disabled,
1 - block sound's,
2 - block text,
3 - block text and sound's. Coz this is very noisy.

**Comment by SamVanheer** [comment]
The `SendAudio` network message handles radio messages so the message handler for that would be a good place to put this.

**Comment by Mistrick** [comment]
Already exist server cmd `ignorerad`. Just write it in client console.

**Comment by RauliTop** [comment]
> Already exist server cmd `ignorerad`. Just write it in client console.

Yes, but will be better to have a client command like @afwn90cj93201nixr2e1re said
